tests/python_args_mixed/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, { parser: "python" });
```

Comment:
There was a change in #3356 so now it should be `run_spec(__dirname, ["python"]);`